src/config/dark-sites.config

```diff
@@ -222,6 +226,7 @@ disneyplus.com
 distro.tube
 djflame.tech
 dlive.tv
+doc.rust-lang.org/book/
```

Comment:
Please remove, this isn't dark-by-default:

![image](https://user-images.githubusercontent.com/25481501/217876637-1e7256fe-410d-41e3-a2e0-5c0e3bab1913.png)

Reply:
Oh, my bad, I must have forgot to disable Dark Reader while performing the test :sweat_smile: